tests/test_data/study_dedicated_schema/manifest.toml

```diff
@@ -0,0 +1,13 @@
+study_prefix = "study_dedicated_schema"
+
+[table_builder_config]
+file_names = ["module1.py", "module1.py", "module2.py"]
+
+[sql_config]
+file_names = ["test.sql"]
+
+[export_config]
+export_list = ["study_dedicated_schema__table_raw_sql"]
+
+[options]
+use_dedicated_schema="dedicated"
```

Comment:
nit: I might like to see a beefier test here - like actually checking that the resulting stuff gets placed in a dedicated schema and the sql is replaced etc. This test currently smoketests that we don't blow up when this option is set, but not that it actually changes anything, yeah?

Reply:
yeah, fair point